### ISSUE TITLE
Questions about training effects

### ISSUE THREAD

**Issue by wangchuan199803** [qgao21/CoreDiff#3]
Hello, I used this model on a dataset that only contains mayo data, and I did not use adjacent slices during training. The training set size is about 1500, and the batch_size size is 32. But the result was very bad, it could even be said to be a disaster. And I observed that the loss function differs greatly on different batches of data. Moreover, the value of the loss function is very, very small. I feel that the model has made no progress during the training process.

**Comment by kklay11** [comment]
win上能跑吗xd

**Comment by wangchuan199803** [comment]
可以跑的，只要显存够用



---原始邮件---
发件人: "Deng ***@***.***&gt;
发送时间: 2024年6月12日(周三) 晚上7:57
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [qgao21/CoreDiff] 终于有个能跑的了 (Issue #3)




 
win上能跑吗xd

**Comment by kklay11** [comment]
谢谢

**Comment by dhyuk54** [comment]
> win上能跑吗xd

你运行成功了吗？

**Comment by 17860286392** [comment]
> 可以跑的，只要显存够用
> […](#)
> ---原始邮件--- 发件人: "Deng ***@***.***&gt; 发送时间: 2024年6月12日(周三) 晚上7:57 收件人: ***@***.***&gt;; 抄送: ***@***.******@***.***&gt;; 主题: Re: [qgao21/CoreDiff] 终于有个能跑的了 (Issue #3) win上能跑吗xd — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

您好，我不太理解代码的顺序，能分享一下您的代码吗，谢谢您

**Comment by Riveer01** [comment]
您好，请问您是怎样准备数据集的呢
